.prout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "checkpoints": {
+    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
```

Comment:
That great big exception is caused by a missing `APPLICATION_SECRET` environment variable, which you need to configure in Heroku. Apologies for this, I've only just [added that required field](https://github.com/guardian/prout/commit/70714e4) to the Deploy to Heroku button - so you can manually add that field to your Heroku app:

![image](https://cloud.githubusercontent.com/assets/52038/10636026/b06e7cfc-77f4-11e5-8f99-1b6d7d474d66.png)

...or deploy a new one.